Dockerfile

```diff
@@ -23,18 +25,24 @@ RUN \
 RUN sed -i '/# Development/,$d' requirements.txt
 RUN apk add --no-cache --virtual .build-deps gcc musl-dev postgresql-dev zlib-dev jpeg-dev libwebp-dev openssl-dev libffi-dev cargo openldap-dev python3-dev && \
     echo -n "INPUT ( libldap.so )" > /usr/lib/libldap_r.so && \
-    python -m venv venv && \
-    /opt/recipes/venv/bin/python -m pip install --upgrade pip && \
-    venv/bin/pip install wheel==0.42.0 && \
-    venv/bin/pip install setuptools_rust==1.9.0 && \
-    venv/bin/pip install -r requirements.txt --no-cache-dir &&\
+    python -m pip install --upgrade pip && \
+    pip install wheel==0.42.0 && \
+    pip install setuptools_rust==1.9.0 && \
+    pip install -r requirements.txt --no-cache-dir &&\
     apk --purge del .build-deps
 
 #Copy project and execute it.
 COPY . ./
 
+# collect the static files
+RUN python manage.py collectstatic_js_reverse
+RUN python manage.py collectstatic --noinput
+# copy the collected static files to a different location, so they can be moved into a potentially mounted volume
+RUN rm -rf /opt/recipes/staticfiles/*
+RUN mv /opt/recipes/staticfiles /opt/recipes/staticfiles-collect
```

Comment:
doesn't this delete the staticfiles first and then tries to move them, the folder should be empty at that point

Reply:
Yes that was a wrong fix for a problem I encountered. This should've been in the boot.sh file. I have fixed this though.